pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/gardener/gardener/pkg/resourcemanager/controller/garbagecollector/references"
 	appsv1 "k8s.io/api/apps/v1"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	batchv1 "k8s.io/api/batch/v1"
 	coordinationv1 "k8s.io/api/coordination/v1"
```

Comment:
Should not we add package for cronjobs? I see, it's used later for clusterroles:
`batchv1beta1 "k8s.io/api/batch/v1beta1"`

Reply:
Done